style.css

```diff
@@ -29,6 +34,81 @@ img {
     margin-bottom: 0.5em;
 }
 
+body svg {
+    box-sizing: initial;
+}
+
+/* navbar */
+
+nav {
+    background-color: var(--white)
+}
+
+nav svg {
+    font-size: inherit;
+    fill: var(--primary-gray);
+    width: 1em;
+}
+
+.search-input {
+    width: 50vw;
+}
+
+nav a {
+    text-decoration: none;
+    color: var(--primary-gray);
+}
+
+img {
+    margin: 0;
+    padding: 0;
+}
+
+.navbar_divider {
+    background-color: var(--primary-gray);
+    height: 32px;
```

Comment:
Why 32px and not 100%? Hardcoding values is usually not a good idea.

Reply:
100% not working. changed it to 2em